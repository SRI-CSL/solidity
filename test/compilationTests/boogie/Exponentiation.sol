pragma solidity ^0.4.23;

contract SomeContract {
    function someFunc() {
        uint x = 10 ** 20 + 1;
        uint y = 100000000000000000001;
        assert(x == y);
    }
}