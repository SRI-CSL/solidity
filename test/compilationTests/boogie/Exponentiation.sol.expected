Verifying SomeContract::someFunc: OK
No errors found.
